src/docs/development/packages-and-plugins/androidx-compatibility.md

```diff
@@ -68,8 +68,61 @@ Use the following instructions:
 
 #### Not recommended: Manually migrate your app
 
+1. In `android/gradle/wrapper/gradle-wrapper.properties` change the line starting with `distributionUrl` like this:
+
+`distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip`
+
+2. In `android/build.gradle`, replace:
+
+```gradle
+dependencies {
+    classpath 'com.android.tools.build:gradle:3.2.1'
+}
+```
+
+by
+
+```gradle
+dependencies {
+    classpath 'com.android.tools.build:gradle:3.3.0'
+}
+```
+
+3. In `android/gradle.properties`, append
+
+```
+android.enableJetifier=true
+android.useAndroidX=true
+```
+
+4. In `android/app/build.gradle`:
+
+Replace version `27` by `28` (`compileSdkVersion` and `targetSdkVersion` under `android {`).
+
+Also, replace
+
+`testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"`
+
+by
+
+`testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"`
+
+Finally, under `dependencies {`, replace
+
+```gradle
+androidTestImplementation 'com.android.support.test:runner:1.0.2'
+androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+```
+
+by
+
+```gradle
+androidTestImplementation 'androidx.test.runner:1.1.1'
+androidTestImplementation 'andoridx.test.espresso:espresso-core:3.1.1'
+```
+
 See [Migrating to
```

Comment:
I would keep this section towards the top still and emphasize that this is the definitive source of truth for how to migrate, but below are some suggested steps which will apply to most (but not all) cases. Not sure of the best phrasing. @Sfshaza WDYT?

>See *Migrating to AndroidX*(link) for detailed instructions on how to do this. Below are some steps that you'll likely need to go through as part of this process, listed here for reference. However the specific things you need to do will depend on your build configuration and could differ from the example changes suggested here.

Reply:
note the typo here:
```
androidTestImplementation 'andoridx.test.espresso:espresso-core:3.1.1'
```

should be:
```
androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
```

(`andoridx` instead of `androidx`)